OpenSim/Simulation/Test/testForces.cpp

```diff
@@ -1670,3 +1677,20 @@ void testExternalForce()
             ASSERT_EQUAL(def, val, 10*accuracy);
     }
 }
+
+void testSerializeDeserialize() {
```

Comment:
This isn't the test I would have expected. An old model is read in (which will indeed trigger the new `updateFromXMLNode()` method), then the file is written and read in again. The `ASSERT()` is checking that serializing a v30508 model and then deserializing it results in the same `ForceSet`—but correctness of the new `updateFromXMLNode()` method is not checked. Perhaps use a model that contains muscles that are both enabled and disabled, and confirm that the new `appliesForce` property is being set correctly?